src/main/scala/junctions/addrmap.scala

```diff
@@ -121,6 +119,12 @@ class AddrMap(
     }.flatten.sortBy(_.region.start)
   }
 
+  flatten.combinations(2) foreach {
+    case (Seq(AddrMapEntry(an, ar), AddrMapEntry(bn, br))) =>
```

Comment:
This is sort of nitpicky. But if you aren't using `an` and `bn`, you can just match on `_`.